clash-prelude/src/Clash/Signal.hs

```diff
@@ -395,29 +400,17 @@ components such as PPLs and 'resetSynchronizer':
 
 @
 topEntity
-  :: Clock System
-  -> Reset System
-  -> Enable System
-  -> Signal System Int
-  -> Signal System Int
-topEntity clk rst =
-  let (pllOut,pllStable) = 'Clash.Intel.ClockGen.altpll' (SSymbol \@\"altpll50\") clk rst
-      rstSync            = 'resetSynchronizer' pllOut ('unsafeToAsyncReset' pllStable)
-  in  'exposeClockResetEnable' f pllOut rstSync
-@
-
-or, using the alternative method:
-
-@
-topEntity2
-  :: Clock System
-  -> Reset System
-  -> Signal System Int
-  -> Signal System Int
-topEntity2 clk rst =
-  let (pllOut,pllStable) = 'Clash.Intel.ClockGen.altpll' (SSymbol \@\"altpll50\") clk rst
-      rstSync            = 'resetSynchronizer' pllOut ('unsafeToAsyncReset' pllStable)
-  in  'withClockReset' pllOut rstSync f
+  :: Clock  System
+  -> Reset  System
+  -> Signal System Bit
+  -> Signal System (BitVector 8)
+topEntity clk rst ena key1 =
+    let  (pllOut,pllStable) = altpll (SSymbol @"altpll50") clk rst
+         rstSync            = 'resetSynchronizer' pllOut (unsafeToHighPolarity pllStable) ena
+    in   exposeClockResetEnable leds pllOut rstSync enableGen
+  where
+    key1R  = isRising 1 key1
+    leds   = mealy blinkerT (1, False, 0) key1R
```

Comment:
Why drop the documentation for the alternative method that uses `withClockResetEnable` ?

Reply:
Hmm, I copied this from somewhere else because the current documentation is broken. I simply forgot to rebuild the second one!

Reply:
..and to escape some things it seems. I pray we can get Markdown support some time.